core/winnt/src/TWinNTSystem.cxx

```diff
@@ -4319,10 +4319,12 @@ const char *TWinNTSystem::GetLibraries(const char *regexp, const char *options,
          // full path not found, so split it to extract the library name
          // only, set its extension to '.lib' and add it to the list of
          // libraries to search
-         _splitpath(str.c_str(), drive, dir, fname, ext);
-         std::string libname(fname);
-         libname += ".lib";
-         all_libs.push_back(libname);
+         if (str.find("C:\\WINDOWS\\") == std::string::npos) {
```

Comment:
Isn't there a Windows API to see what the Windows install folder is, or maybe even whether a DLL / path is a system one?

Reply:
Good point, thanks!